notfound/extension.py

```diff
@@ -49,11 +49,13 @@ def pathto(otheruri, resource=False, baseuri=None):
         if not resource:
             otheruri = app.builder.get_target_uri(otheruri)
 
-        if baseuri is None:
+        if baseuri is None and not app.config.notfound_gh_pages:
```

Comment:
I'm not sure if we want to call this too specific as `gh_pages` or maybe something more general since it's not only useful for GH pages but for any other host where the docs are mounted at `/`.

Reply:
Perhaps something like `notfound_no_prefix`?

Reply:
Yes. Maybe adding `url` somewhere in the name: `notfound_no_urls_prefix`.

Reply:
Resolved by 46c4021